src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -194,6 +194,30 @@ define(function (require, exports, module) {
             skipCallback = true;
             CommandManager.execute("bramble.addCodeSnippet", args[0]).always(callback);
             break;
+        case "CODEMIRROR_RELAY":
+            var editor = EditorManager.getActiveEditor();
```

Comment:
+1 to this. Have a separate file in the same folder as `Editor.js` (maybe `Collaboration.js`) and put all this code in there. Then just require it in `Editor.js` and call a function to initialize it or something like that.